packages/dredd/scripts/smoke.sh

```diff
@@ -2,52 +2,46 @@
 # Performs a smoke test verifying the package produced by the build
 # is installable and that Dredd's core life functions are okay
 
-
 # Aborts as soon as anything returns non-zero exit status
 set -e
 
+TMPDIR="$(mktemp -d)"
+PROJECT_DIR="$(pwd)/../.."
+
+# install monorepo package into $TMPDIR
+install() {
+  PACKAGE="$1"
+
+  cd "$PROJECT_DIR/packages/$PACKAGE"
+  TARBALL="$(npm pack | tail -n1)"
+  mv "$PROJECT_DIR/packages/$PACKAGE/$TARBALL" "$TMPDIR"
+
+  cd "$TMPDIR" && npm install --no-save "$TARBALL"
+}
+
+git clone https://github.com/apiaryio/dredd-example.git "$TMPDIR"
+install dredd-transactions
+install dredd
 
-if [ ! -z "$CIRCLECI" ]; then
```

Comment:
I've removed the requirement that smoke tests are only able to be ran on CircleCI. It made it harder to test and I didn't see the need to disallow running this locally.

Reply:
I think the historical reason to forbid that is due to smoke test performs file-system operations that may break something, if run on local. However, during my debugging of the smoke test there hasn't been such occasions, so I support removing this check.

Reply:
Yeah, and I've moved to using a tempoary directory so we shouldn't leave any  state around anymore.